### DISCUSSION TITLE
/iptv-org/epg

### DISCUSSION THREAD

**Discussion by olek52** [iptv-org/community#312] [Q&A] [answered]
https://github.com/iptv-org/epg   I was looking for an epg for Germany uploaded   I added   [magentatv.at](https://magentatv.at/)    and noticed that the offer from  https://web.magentatv.de/tv-guide  ..   magentatv.at  differs none for example  DAZN 1  DAZN 2 .So my question is it possible to add versions  magentatv.de    https://web.magentatv.de/tv-guide  Thank you

**Comment by olek52** [comment]
sorry for my english google translate

https://github.com/iptv-org/epg I was looking for an epg for germany, uploaded magentatv.at and noticed the offer is
    different than here https://web.magentatv.de/tv-guide ..w magentatv. no example DAZN 1 DAZN 2. So my question is it possible to add magentatv.de versions https://web.magentatv.de/tv-guide Thank you

**Comment by BellezaEmporium** [comment, marked as answer]
It is quite understandable, no worries about your english.

If you're talking about missing channels, then you'll need to ask through the epg repository. You're actually in the iptv repository.

**Comment by olek52** [comment]
ok   thx   I was looking for any contact

**Comment by olek52** [comment]
https://iptv-org.github.io/   I was looking for channels   V Sport 1 Norway ,  V Sport 2 Norway   V Sport 3 Norway    V Sport + Norway 
and info that it's an epg  https://iptv-org.github.io/epg/guides/no/allente.se.xml    added to  TiviMate    epg  I manually search for these channels by name and I don't see them?   V Sport 1 Norway _ V Sport 2 _ V Sport 3 _ V Sport +

**Reply by BellezaEmporium** [reply]
>  https://iptv-org.github.io/   I was looking for channels   V Sport 1 Norway ,  V Sport 2 Norway   V Sport 3 Norway    V Sport + Norway 
> and info that it's an epg  https://iptv-org.github.io/epg/guides/no/allente.se.xml    added to  TiviMate    epg  I manually search for these channels by name and I don't see them?   V Sport 1 Norway _ V Sport 2 _ V Sport 3 _ V Sport +

Channels that are in the EPG doesn't mean that they are available in our playlists. Especially the Sports channels, that are DMCAd most of the time.

**Comment by olek52** [comment]
I have channels, but in the Tivimate application you can manually assign an epg to a channel, just search for a given channel in the epg assigned in this application, only there are no such names in this epg   V Sport 1 Norway _ V Sport 2 _ V Sport 3 _ V Sport +


https://s9.gifyu.com/images/photo_2023-01-18_11-23-42.jpg     postscript  epg_se it's your epg
